docs/source/using/troubleshooting.adoc

```diff
@@ -11,6 +11,26 @@ toc::[]
 
 This section contains solutions to common problems that you might encounter while using Minishift.
 
+[[minshift-update-failed-due-to-permission-denied]]
+== Permission cause update to fail
+
+Depending on your operating system if you have minishift binary installed in location where current user does not have write permission then `minishift update` will fail.
+
```

Comment:
> o if the user does not have access permission on the file system to replace the minishift binary then the update will fail.

That's not quite correct. The user can have permissions to the binary and the update will still fail, if the directory the binary lives in is not writable.